ocsp/updater/updater.go

```diff
@@ -111,7 +142,9 @@ func New(
 	issuers []rocsp_config.ShortIDIssuer,
 	serialSuffixes []string,
 	ogc capb.OCSPGeneratorClient,
-	config ocsp_updater_config.Config,
+	// A temporary evil. This constructor should not take a JSON config as input;
```

Comment:
Should have a TODO unless you fix before merging :-)

Reply:
This one is actually just a replication of the [six year old](https://github.com/letsencrypt/boulder/commit/2d0dee4ce1defe3e2fae17e4d2c92d6b4a07fa6e) comment in main.go:
https://github.com/letsencrypt/boulder/blob/3b9f3dc0006e335016e3993fc62d5720c6c60b52/cmd/ocsp-updater/main.go#L147-L148

Not sure if this is something I'll get to as part of this refactor; if not, I'll definitely file a ticket for it as followup.